iModelJsNodeAddon/IModelJsNative.cpp

```diff
@@ -2807,12 +2807,55 @@ struct NativeGeoServices : BeObjectWrap<NativeGeoServices>
         return results;
         }
 
+    static Napi::Value GetListOfGCS(NapiInfoCR info)
+        {
+        REQUIRE_ARGUMENT_BOOL(0, ignoreLegacy);
+        OPTIONAL_ARGUMENT_ANY_OBJ(1, extent, Napi::Object::New(info.Env()));
+
+        // Check if parameter is present and valid
+        bool extentIsValid =    extent.Get("low").IsObject() && 

```

Comment:
You are not handling all of the possible representations of `Range2dProps`. Use `BeJsGeomUtils::DRange2dFromJson`.

Reply:
Done

Reply:
> Done

How so? You're not calling `DRange2dFromJson` and you're not handling all the possible `Range2dProps` representations.

All you need to do is: if `extent` is provided, pass it directly to `BeJsGeomUtils::DRange2dFromJson`. You don't need to parse any of its fields yourself.

Reply:
My bad. Forgot to push my commits yesterday, sorry :( 
BTW, to check if the extent is provided, I did not find any other way than check if it is equal to (0, 0, 0, 0). I guess there must be another way ?